### ISSUE TITLE
Exploring Data Frames: Downloading gapminder files before reading them in?

### ISSUE THREAD

**Issue by RaoulWolf** [swcarpentry/r-novice-gapminder#187]
I noticed that after Challenge 1 in Lesson 5 (Exploring Data Frames), the gapminder dataset is supposed to be read in, either from the data subfolder, or directly from the URL. However, I would suggest downloading the data set first (not mentioned in the material at this moment!) and then go for option 1. In an existing RStudio project, the code could look something like this:

```
download.file("https://raw.githubusercontent.com/resbaz/r-novice-gapminder-files/master/data/gapminder-FiveYearData.csv", destfile = "data/gapminder-FiveYearData.csv")
gapminder <- read.csv("data/gapminder-FiveYearData.csv")
```

In my opinion this is good practice in order to have your data set available locally, and it also fixes the wrong gapminder URL.

**Comment by naupaka** [comment]
@RaoulWolf yes I think we agree that would be a better approach. #172, #173, and this (#187) are all related to the same problem. I think a fix would include changing the path to the dataset to a URL for the gapminder website in episode 1, as suggested by @ashander, adding the `download.file()` line before the `read.csv()` line in episode 5 as suggested above and in #172. PRs welcome to fix these.

**Comment by lmsimp** [comment]
Hi @naupaka, @RaoulWolf  I have attempted to address the above issue in this [pull request](https://github.com/swcarpentry/r-novice-gapminder/pull/204) as part of my SWC checkout part I and also issue #173 related to the above.

**Comment by naupaka** [comment]
Closed by #204
